code/datums/mutations/body.dm

```diff
@@ -394,3 +394,78 @@
 		return //remove the 'edge' cases
 	to_chat(owner, "<span class='danger'>You trip over your own feet.</span>")
 	owner.Knockdown(30)
+
+/datum/mutation/human/martyrdom
+	name = "Internal Martyrdom"
+	desc = "A mutation that makes the body destruct when near death. Not damaging, but very, VERY disorienting."
+	locked = TRUE
+	quality = POSITIVE //not that cloning will be an option a lot but generally lets keep this around i guess?
+	text_gain_indication = "<span class='warning'>You get an intense feeling of heartburn.</span>"
+	text_lose_indication = "<span class'notice'>Your internal organs feel at ease.</span>"
+
+/datum/mutation/human/martyrdom/on_acquiring()
+	RegisterSignal(owner, COMSIG_MOB_STATCHANGE, .proc/bloody_shower)
+
+/datum/mutation/human/martyrdom/on_losing()
+	UnregisterSignal(owner, COMSIG_MOB_STATCHANGE)
+
+/datum/mutation/human/martyrdom/proc/bloody_shower(new_stat)
+	if(new_stat != UNCONSCIOUS)
+		return
+	var/list/organs = owner.getorganszone(BODY_ZONE_HEAD, 1)
+
+	for(var/obj/item/organ/I in organs)
+		I.Remove(owner, TRUE)
+
+	explosion(get_turf(owner), 0, 0, 2, 0, TRUE)
+	for(var/mob/living/carbon/human/H in range(2,owner))
+		var/obj/item/organ/eyes/eyes = H.getorganslot(ORGAN_SLOT_EYES)
+		to_chat(H, "<span class='userdanger'>You are blinded by a shower of blood!</span>")
+		H.Stun(20)
+		H.blur_eyes(20)
+		eyes?.applyOrganDamage(5)
+		H.confused += 3
+	for(var/mob/living/silicon/S in range(2,owner))
+		to_chat(S, "<span class='userdanger'>Your sensors are disabled by a shower of blood!</span>")
+		S.Paralyze(60)
+	owner.gib()
+
+/datum/mutation/human/headless
+	name = "H.A.D.S."
+	desc = "A mutation that makes the body reject the head. Stands for Head All Depleted Syndrome. Warning: Removing this mutation is very dangerous."
+	quality = NEGATIVE //holy shit no eyes or tongue or ears
+	text_gain_indication = "<span class='warning'>Something feels off.</span>" //heh
+
+/datum/mutation/human/headless/on_acquiring()
```

Comment:
headless mobs are very buggy, as evidenced by dulluhans.

Strongly against this mutation

Reply:
![image](https://user-images.githubusercontent.com/40974010/74788739-766a2b80-5267-11ea-91e8-d728663f5497.png)

(tl;dr: most issues are with dullahands exclusively, i'll fix the issue that isn't)

Reply:
maybe.

Reply:
![image](https://user-images.githubusercontent.com/40974010/74794898-580d2b80-5279-11ea-8a3e-641b844cc7c0.png)
hardcrit last breath seems to still be working for me

Reply:
This is fine, most dullahan issues are with the nonsense relay stuff. This was supported for changelings for a while.